config/application.yml

```diff
@@ -0,0 +1,3 @@
+SENDGRID_PASSWORD: zx0wd0ff
```

Comment:
You don't want this file appearing in your source control. You may have added it before you changed your `.gitignore` file.